### DISCUSSION TITLE
TTS responses with word alignment

### DISCUSSION THREAD

**Discussion by rose-m** [deepgram/community#894] [General help] [answered]
Hey there,

I wanted to reach out and ask if there's a way to get word alignment information for the generated audio from the TTS API? So when providing "This is a test" we'd like to know at which point in the audio "This" is basically spoken (start / end) - ideally this would be on individual character level.

I didn't find that info anywhere in the docs for now... Mabye you can help me out there?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @rose-m, we don't currently offer word alignment info on our TTS output audio.

If word timings are essential, you can transcribe the audio with our pre-recorded STT, which does provide word-level timestamps.

However, that would incur an additional transcription cost, which may not be acceptable for your use case.

**Reply by rose-m** [reply]
Thank you for your response - is that on the timeline / planned as a feature?

**Reply by jkroll-deepgram** [reply]
Hi @rose-m, we have heard that request from customers, and while it's not scoped out on our roadmap, it is a feature we are considering adding in the future. I've logged your interest on that feature, so we can reach out when we have additional information on our plans.

**Reply by PeachyCard** [reply]
Right now we are using 11Labs because they support timestamps on TTS.  If you had this functionality, then we would leave 11Labs as your latency is much better

**Reply by akshaymanathkar** [reply]
Is this feature getting added soon or still not on the roadmap?

**Reply by flazouh** [reply]
Would appreciate to add it.

**Comment by kerenduchanclinch** [comment]
Hi @jkroll-deepgram, any updates on this issue? We are also in the same boat as rose-m

**Comment by schthms** [comment]
Any updates on this? Would be nice to have it speech + timestamps in one go like with Cartesia

**Reply by naomi-deepgram** [reply]
Hiya! Let me ask around internally and see if I have any updates for ya~

**Reply by flazouh** [reply]
> Hiya! Let me ask around internally and see if I have any updates for ya~

Any news @naomi-deepgram ?
